web/client/src/library/components/graph/help.ts

```diff
@@ -149,55 +127,66 @@ function getNodesAndEdges({
     }
   }
 
-  return {
-    edges: outputEdges,
-    nodes: Object.values(nodesMap),
-    nodesMap,
-  }
+  return outputEdges
 }
 
 function getNodeMap({
-  modelNames,
   lineage,
-  highlightedNodes,
   models,
-  sources,
-  model,
   withColumns,
 }: {
-  modelNames: string[]
-  lineage: Record<string, Lineage>
-  highlightedNodes: Record<string, string[]>
   models: Map<string, Model>
-  sources: Set<string>
-  model: ModelSQLMeshModel
   withColumns: boolean
+  lineage?: Record<string, Lineage>
 }): Record<string, Node> {
+  if (isNil(lineage)) return {}
+
   const NODE_BALANCE_SPACE = 64
   const COLUMN_LINE_HEIGHT = 24
   const CHAR_WIDTH = 8
   const MAX_VISIBLE_COLUMNS = 5
 
-  return modelNames.reduce((acc: Record<string, Node>, label: string) => {
+  const currentModelNames = Array.from(models.keys())
+  const lineageAllModels = Object.values(lineage)
+    .map(({ models }) => models)
+    .flat()
+  const modelNames = Object.keys(lineage)
+  const sources = new Set(
+    Object.values(lineage)
+      .map(l => l.models)
+      .flat(),
+  )
+
+  return modelNames.reduce((acc: Record<string, Node>, modelName: string) => {
+    const model = models.get(modelName)
     const node = createGraphNode({
-      label,
-      type: models.has(label) ? 'model' : 'cte',
+      label: modelName,
+      type: isNotNil(model)
+        ? (model.type as LineageNodeModelType)
+        : // If model name present in lineage but not in global models
+        // it means either this is a CTE or  model is UNKNOWN
```

Comment:
Is an UNKNOWN model the same as an EXTERNAL model?
```suggestion
        // it means either this is a CTE or model is UNKNOWN
```

Reply:
User can define external models (like models in schema.yaml) but case for UNKNOWN is when for example user misspelled model name
![Screenshot 2023-08-08 at 10 31 36 AM](https://github.com/TobikoData/sqlmesh/assets/5644753/233ba1d7-5afc-4455-9832-06bb98c5db60)